server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -0,0 +1,736 @@
+The count of login failures also will not be accurate with the `ASYNC` strategy.
+
+For the environments with slower network between data centers and probability of DoS, it is recommended to not backup the `loginFailures` cache at all.
+
+* It is recommended to keep the `sessions` and `clientSessions` caches in `SYNC`. Switching them to `ASYNC` is possible only if you are sure that user requests and backchannel requests (requests from client applications to {project_name} as described link:requestprocessing[here]) will be always processed on same site. This is true, for example, if:
+** You use active/passive mode as described link:modes[here].
+** All your client applications are using the {project_name} link:http://www.keycloak.org/docs/latest/securing_apps/index.html#_javascript_adapter[Javascript Adapter]. The Javascript adapter sends the backchannel requests within the browser and hence they participate on the browser sticky session and will end on same cluster node (hence on same site) as the other browser requests of this user.
+** Your load balancer is able to serve the requests based on client IP address (location) and the client applications are deployed on both sites.
++
+For example you have 2 sites LON and NYC. As long as your applications are deployed in both LON and NYC sites too, you can ensure that all the user requests from London users will be redirected to the applications in LON site and also to the {project_name} servers in LON site. Backchannel requests from the LON site client deployments will end on {project_name} servers in LON site too. On the other hand, for the American users, all the {project_name} requests, application requests and backchannel requests will be processed on NYC site.
++
+* For `offlineSessions` and `offlineClientSessions` it is similar, with the difference that you even don't need to backup them at all if you never plan to use offline tokens for any of your client applications.
+
+Generally, if you are in doubt and performance is not a blocker for you, it's safer to keep the caches in `SYNC` strategy.
+
+WARNING: Regarding the switch to SYNC/ASYNC backup, make sure that you edit the `strategy` attribute of the the `backup` element. For example like this:
+```
+<backup site="site2" failure-policy="FAIL" strategy="ASYNC" enabled="true">
+```
+Not the `mode` attribute of cache-configuration element.
```

Comment:
Not the `mode` attribute of cache-configuration element.
->
Note the `mode` attribute of `cache-configuration` element.